setup.py

```diff
@@ -349,6 +353,7 @@ def do_setup():
             'azure_data_lake': azure_data_lake,
             'azure_cosmos': azure_cosmos,
             'azure_container_instances': azure_container_instances,
+            'azure_hdinsight': azure_hdinsight,
```

Comment:
I am doubtful that we should separate this for every single service from Azure. I am afraid that the list of possible options will explode in a moment. Google has only one option - 'gcp_api'.